tests/test_writer.py

```diff
@@ -1637,9 +1636,6 @@ def test_watermarking_reportlab_rendering(tmp_path):
     base_page.merge_page(watermark)
     writer.add_page(base_page)
 
-    for page in writer.pages:
-        page.compress_content_streams()
```

Comment:
I removed those lines to have a more concise test case.
Those calls have zero impact visually on the resulting PDF document